cloudinit/sources/DataSourceRbxCloud.py

```diff
@@ -71,11 +71,13 @@ def gratuitous_arp(items, distro):
 
 
 def get_md():
-    rbx_data = None
+    """Returns False (not found or error) or a dictionary with metadata."""
```

Comment:
instead of (or in addition to) this long-winded doc-string, we could add a type annotation

Reply:
> instead of (or in addition to) this long-winded doc-string, we could add a type annotation

The return type is `Union[dict, bool]` which we can't express in our still-supporting-3.4 codebase.